logster/outputs/dogstatsd.py

```diff
@@ -0,0 +1,55 @@
+from logster.logster_helper import LogsterOutput
+import re
+import socket
+
+
+class DogstatsdOutput(LogsterOutput):
+    shortname = 'dogstatsd'
+
+
```

Comment:
nitpick: remove extra blank line and in two places below. only single line between methods in a class